docs/_static/data/supported_versions.json

```diff
@@ -2,9 +2,9 @@
   "data": [
         {
         "version": "ScyllaDB 2025.2",
-        "released": "Pre-release",
-        "status": "Next",
-        "end_of_life": "According to the policy",
+        "released": "July 2025",
+        "status": "Supported",
+        "end_of_life": "After 2027.1 or 2025.4 is released",
```

Comment:
According to our policy 2026.2 will make 2025.2 EOL

Reply:
I thought 2025.3 / 2025.4 will, no?